javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,42 @@
+//A future age calculator
+const yearOfBirth = 1986
+const yearFuture = 2027
+my = yearFuture - yearOfBirth
+console.log("You will be " + my + " in 2027")
+
+//A dog age calculator 
+const dogAge = 1957
```

Comment:
`const dogYearOfBirth = 1957;`
Please check the assignment again https://github.com/HackYourFuture-CPH/JavaScript/blob/main/javascript1/week1/homework.md